katalog/prometheus-adapter/apiservice.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apiregistration.k8s.io/v1beta1
```

Comment:
It's a good sanity check for yaml files to have this as begining of the file. `---`
https://yaml.org/spec/1.0/#id2561718

I use **yamllint**.  Yaml syntax are not the place to be confortable debugging.

Reply:
totally agree @iknite, my OCD is getting worst everytime i see a yaml without `---`